spectrumx_visualization_platform/spx_vis/api/views.py

```diff
@@ -1,119 +1,212 @@
 from django.http import FileResponse
-from rest_framework import filters
-from rest_framework import permissions
-from rest_framework import status
-from rest_framework import viewsets
-from rest_framework.decorators import action
-from rest_framework.parsers import FormParser
-from rest_framework.parsers import MultiPartParser
+from rest_framework import filters, permissions, status, viewsets
+from rest_framework.decorators import action, api_view
+from rest_framework.parsers import FormParser, MultiPartParser
+from rest_framework.request import Request
 from rest_framework.response import Response
-
-from spectrumx_visualization_platform.spx_vis.api.serializers import CaptureSerializer
-from spectrumx_visualization_platform.spx_vis.api.serializers import FileSerializer
+from spectrumx.errors import FileError
+from spectrumx.models.captures import CaptureType
+from spectrumx_visualization_platform.spx_vis.api.serializers import CaptureSerializer, FileSerializer
 from spectrumx_visualization_platform.spx_vis.capture_utils.sigmf import SigMFUtility
-from spectrumx_visualization_platform.spx_vis.models import Capture
-from spectrumx_visualization_platform.spx_vis.models import File
+from spectrumx_visualization_platform.spx_vis.models import Capture, File
+
+from datetime import datetime, timedelta
+
+@api_view(["GET"])
+def capture_list(request: Request) -> Response:
+    """Get the list of captures for the current user."""
+    file_list = []
+    sds_file_list = []
```

Comment:
Since files and captures are two distinct objects, and we're dealing with captures here, it would be clearer to call these `capture_list` and `sds_capture_list`